AssetManagementTestProject/TestCases/US308_DisableUserTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+
+namespace AssetManagementTestProject;
+
+[TestFixture]
+public class US308_DisableUserTest : NUnitWebTestSetup
+{
+
+    protected ManageUserPage? ManageUserPage;
+
+    public void TC02_AdminCanCancelDisableAction()
+    {
+        ManageUserPage = new ManageUserPage();
+        LoginPage?.Login(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD);
+        DriverBaseAction?.WaitToBeVisible(HomePage.HeaderHomePage);
+        Asserter?.AssertElementIsDisplayed(HomePage.HeaderHomePage);
+        ManageUserPage.GoToUserList();
+        ManageUserPage.InputSearch(NewUser.Data.StaffCode);
+        DriverBaseAction?.WaitToBeVisible(ManageUserPage.TableData);
+        Asserter?.AssertElementIsDisplayed(ManageUserPage.TableData);
+        ManageUserPage.SelectDisable();
+        DriverBaseAction?.WaitToBeVisible(ManageUserPage.HeaderDisableUser);
```

Comment:
I think header verification is enough already

Reply:
Done